### DISCUSSION TITLE
Additional student list filter & sort options

### DISCUSSION THREAD

**Discussion by oy3o** [lonqie/SchaleDB#40] [Ideas and Suggestions]
Calculate the attribute ranking of the current level and equipment 
Calculate the attribute ranking of the full-level and full equipment.
Skill Screening Omissions: such as Cherino Cost Regen was not found in the screening.
Increase the screening of debuffs such as Armor Break.
Recommend suitable armor-piercing candidates based on monster attributes such as high defense.

**Comment by lonqie** [comment]
Thanks for the suggestions.

As of the update b7ef57cc758f99e568a6b6441a26e9724155b49e, sort options for stats now use the calculated maximum, and the option has been added to use collection saved stats instead.

The sub skill filter only considers the buffs from Special students, so Cherino being excluded from the Cost Recovery filter is the expected result. Filter name has been changed slightly so it is more clear.

I am not sure I understand your last two suggestions, do you mean you want a filter for students that can inflict specific debuffs, such as DEF down? This does already exist when selecting debuffs to apply to a target.
